test/ExtensibleEvents.test.ts

```diff
@@ -104,6 +114,8 @@ describe('ExtensibleEvents', () => {
             event = extev.parse(input);
             expect(event).toBeDefined();
             expect(event instanceof MyCustomEvent).toBe(true);
+            expect(event.serialize().type).toBe("myNamespace.name");
+            expect(event.serialize().content).toHaveProperty("hello", "world");
```

Comment:
not really sure what these and below are testing - `serialize()` is ultimately a function of our mock ExtensibleEvent, so it feels like we're testing that we wrote the mock correctly. It feels a bit more important to ensure that the code isn't calling serialize needlessly, so having the throw.

Reply:
an `expect` in the constructor would probably make sense, yea. Also would apply to the `MockEvent` class.

as a minor nitpick in this area: `org.example` namespace please, like `org.example.custom` :)

Reply:
Do you mean `myNamespace`?  I'm 99% sure I copied that from you!

Reply:
`myNamespace` is a variable which maps to an unstable-only value: `const myNamespace = new UnstableValue(null, "org.example.custom.event");` ;)

Reply:
Ah sorry, really dumb!